### ISSUE TITLE
Swap failed, unable to redeem my BTC. 

### ISSUE THREAD

**Issue by MavGan12** [UnstoppableSwap/unstoppableswap-gui#192]
Swap failed at step 3 or 4, not really sure. Now i'm unable to redeem my BTC. Are my coins lost?

Im on MacOS. Tried the solution i read in other threads but wasn't able to make it work on MacOS (https://github.com/UnstoppableSwap/unstoppableswap-gui/issues/147#issuecomment-1647742247)

## Steps to Reproduce

[debug log.docx](https://github.com/UnstoppableSwap/unstoppableswap-gui/files/14815896/debug.log.docx)

## Your Environment

- Node version : UnstoppableSwap 0.5.3
- Operating System and version :  Macbook Pro M1, MacOS Ventura

**Comment by binarybaron** [comment]
Please see this comment https://github.com/UnstoppableSwap/unstoppableswap-gui/issues/190#issuecomment-2018132638

Has your swap gone through?

**Comment by MavGan12** [comment]
Nope, unfortunately not. Keeps failing and says "cancel time lock has expired". Also failing when trying the option "attempt manual cancel and refund". Is there any solution to refund my btc?

**Comment by binarybaron** [comment]
Please upgrade to the latest pre-release version of the GUI and resuming the swap.
 https://github.com/UnstoppableSwap/unstoppableswap-gui/releases/tag/v0.6.0

**Comment by binarybaron** [comment]
Which state is your swap in? Would you mind sharing the logs of your swap? Feel free tor redact what you like.

**Comment by Ramon44-lab** [comment]
> Nope, unfortunately not. Keeps failing and says "cancel time lock has expired". Also failing when trying the option "attempt manual cancel and refund". Is there any solution to refund my btc?

They will never reply you on here and they can’t do anything I almost lost about 10 ethereum few days ago thanks to blackhathacker who helped me recover my coins from the scammers  message him on telegram https://t.me/Blackhathacker110
